### DISCUSSION TITLE
Show not-recording indicator in Birdseye or don't show streams of not-recording cameras in Birdseye

### DISCUSSION THREAD

**Discussion by kankadev** [blakeblackshear/frigate#11730] [Ideas]
I use PTZ to automatically turn the cameras towards the ceiling or wall when I don't want them to record. This allows me to see directly where the camera is pointing and therefore not recording. I always have Birdseye open on a television. I can also see where the cameras are pointing. However, I have problems with various cameras. They "shift" their presets after a while, so I have to constantly readjust them. I use some automations to control these cameras. When I leave the house, I want the cameras to take up their actual recording position. But since this is faulty, I wanted to use the switches in Home Assistant. I noticed that there is no indicator in Birdseye as to which camera is currently recording and which is not.

I would prefer to see an icon directly in Birdseye to indicate whether recording is currently running. Or a setting that only cameras that are currently recording can be seen in Birdseye wouldn't be bad either.

**Comment by hawkeye217** [comment]
Assuming Birdseye is set to `motion` mode, just have your automation disable recording, detection, and motion detection on the cameras you don't want to appear in Birdseye. 

Cameras with disabled motion detection won't generate motion, and thus won't appear in Birdseye.
